NetKAN/REKT.netkan

```diff
@@ -0,0 +1,19 @@
+{
+  "spec_version": "v1.4",
+  "depends": [
+    { "name": "ModuleManager" },
+    { "name": "RetractableLiftingSurface" }
+  ],
+  "install": [
+    {
+      "install_to": "GameData",
+      "find": "REKT-0.4.3.1",
+      "filter": ["RetractableLiftingSurface.version"]
+    }
+  ],
+  "$vref": "#/ckan/ksp-avc",
```

Comment:
This ensures the correct .version file

    "$vref": "#/ckan/ksp-avc/rekt.version",